lib/logstash/inputs/tcp.rb

```diff
@@ -107,6 +107,7 @@ class LogStash::Inputs::Tcp < LogStash::Inputs::Base
   config :ssl_extra_chain_certs, :validate => :array, :default => []
 
   HOST_FIELD = "host".freeze
+  HOST_IP_FIELD = "[@metdata][ip_address]".freeze
```

Comment:
This should be `"[@metadata]"`. The "a" is missing.

Reply:
@redbaron4 oops thanks for spotting this! Will fix this within the next 2 days once I'm back from vacation.

Reply:
@original-brownbear Looks like this has been fixed in the master (though not yet released)